# Goal
folding_assembly_controller/AdaptiveController adaptive_params
bool use_pose_goal
bool skip_final_alignment
float64 theta_lim
float64 max_contact_force
folding_assembly_controller/PoseGoal pose_goal
folding_assembly_controller/VelocityGoal velocity_goal
---
# Result
---
# Feedback
string phase
float64 current_angle
float64 control_period
